### ISSUE TITLE
setup.py install fails on different pip version

### ISSUE THREAD

**Issue by vivonk** [scorelab/Bassa#466]
## Expected Behavior
On Travis build, setup.py exit with an error because in python3 pip.req is working with pip 9 version. On pip install, the latest version is pip 10 and code should work fine.

## Current Behavior
In pip 10, setup.py exit with following error
```
Traceback (most recent call last):
  File "setup.py", line 6, in <module>
    from pip.req import parse_requirements
ModuleNotFoundError: No module named 'pip.req'
```

## Possible Solution

```
try: # for pip >= 10
    from pip._internal.req import parse_requirements
except ImportError: # for pip <= 9.0.3
    from pip.req import parse_requirements
```

## Your Environment

* Version used: latest one
* Environment name and version: Chrome 64, Node 6.14, Python 3, standard bassa library versions
* Operating System and version (desktop or mobile): MacOS, Linux(Ubuntu)

**Comment by vivonk** [comment]
This issue has been resolved in #468, should we close this?

**Comment by vivonk** [comment]
issue resolved, closing this 😊
